app/dig.go

```
@@ -1,20 +1,53 @@
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
 )
 
 var digContainer *dig.Container = nil
 
 func Setup() {
 	if digContainer == nil {
+
+		if config.AppConfig.IsDevelopment {
+			devLoggerSetup()
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)